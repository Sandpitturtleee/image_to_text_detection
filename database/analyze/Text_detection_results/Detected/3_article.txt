iceminister rozwoW ju Jadwiga EmileWIcZ spotyka się z wiceministrem energii Grzegorzem Tobiszowskim. Temat: walka ze smogicm. Rcsort cncrgiinie chce się zgodzić na tzw. certyfikaty dla węgla Emilewicz łagodnic przckonujcithmaczy. Tobiszowskijestniewzruszony; po swojej stroniema tysiącegórników.Jcdnak Emilcwicz wychodzize spotkaniaimówi:-Bę dą certyfikaty i koniec. Podwóchmicsiącachresort cncrgii odpuszcza. - Bo Emilcwiczniedajesobiew kaszę dmuchać-mówiąjej koledzy. Gdyw niedzielę Ryszard Terleckina TwitterzenapisaŁ zekandydatura Emilcwiczna prczydcn ta Krakowa (zgłoszona przez Jarosława Gowina) nie została uzgodniona z PiS, zarcagowała blyskawicznie: ,Ozgodniena czas zacząć. Zgodzimy się przeciez, żc cpoka brodatych prof wkr skim ma 'gistracie dobiegakon ca:)". Mógłtobyć przytykdo Terlcckicgo; który bcz powodzcnia startowałna prezydenta, ale tez delikatna aluzja do rządzącego czwartą kadencję Jacka chrowskicgo MajNie kryje własnego zdania. Poparła prezydenckie weto ustaw 0 SNiKRS.Zrobiła to zarówno wbrew JarosławowiKaczyńskiemu; od którego raczej się dystansujc, jakiwbrcw bczpośredniemu przełożonemu Mateuszowi Morawieckiemu W20l4r stangła na czclemałopolskich struktur Polski Razem Jarosława Gowina w2015r: została wiccprczcscm partii potem wiceministrem rozwoju. Jej konik to innowacyjność; start-upyi walka ze smogicm Jcstcm z Krakowainicmogę udawać, że problemuniema powtarza Emilewicz Pod Wawclcm z wyróżnicniem skonczyła liceum, a potem politologię na Uniwersytecic Jagiclońskim. Jcsttrochę typem kujona, ale zdystansem do siebie - mówijej wspólpracownik OmiNIKA Wantuch 