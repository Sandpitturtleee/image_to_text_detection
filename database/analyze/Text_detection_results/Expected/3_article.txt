Wiceminister rozwoju Jadwiga Emilewicz spotyka się z wiceministrem energii Grzegorzem Tobiszowskim. Temat: walka ze smogiem. Resort energii nie chce się zgodzić na tzw. certyfikaty dla węgla. Emilewicz łagodnie przekonuje i tłumaczy. Tobiszowski jest niewzruszony, po swojej stronie ma tysiące górników. Jednak Emilewicz wychodzi ze spotkania i mówi: - Będą certyfikaty i koniec. Po dwóch miesiącach resort energii odpuszcza. - Bo Emilewicz nie daje sobie w kaszę dmuchać - mówią jej koledzy. Gdy w niedzielę Ryszard Terlecki na Twitterze napisał, że kandydatura Emilewicz na prezydenta Krakowa (zgłoszona przez Jarosława Gowina) nie została uzgodniona z PiS, zareagowała błyskawicznie: „Uzgodnienia czas zacząć. Zgodzimy się przecież, że epoka brodatych prof. w krakowskim magistracie dobiega końca:)". Mógł to być przytyk do Terleckiego, który bez powodzenia startował na prezydenta, ale też delikatna aluzja do rządzącego czwartą kadencję Jacka Majchrowskiego. Nie kryje własnego zdania. Poparła prezydenckie weto ustaw o SN i KRS. Zrobiła to zarówno wbrew Jarosławowi Kaczyńskiemu, od którego raczej się dystansuje, jak i wbrew bezpośredniemu przełożonemu - Mateuszowi Morawieckiemu. W 2014 r. stanęła na czele małopolskich struktur Polski Razem Jarosława Gowina, w 2015 r. została wiceprezesem partii, a potem wiceministrem rozwoju. Jej konik to innowacyjnośc, start-upy i walka ze smogiem. - Jestem z Krakowa i nie mogę udawać, że problemu nie ma - powtarza Emilewicz. Pod Wawelem z wyróżnieniem skończyła liceum, a potem politologię na Uniwersytecie Jagiellońskim. - Jest trochę typem kujona, ale z dystansem do siebie - mówi jej współpracownik.
